grails-app/assets/javascripts/streama/translations/TK_tk.js

```diff
@@ -0,0 +1,189 @@
+      RATING: 'Meşhurlyk derejesi',
+      STATUS: 'Statusy',
+      STATUS_VALUE: {
+        'completed': 'Tamamlandy',
+        'viewing': 'Görülýär',
+        'unviewed': 'Görülmedik'
+      },
+      VOTES: 'Berlen sesler',
+      OVERVIEW: 'Gysgaça',
+      GENRE: 'Žanr',
+      TRAILER: 'Bildiriş',
+      SEASON: 'Möwsüm',
+      SUBTITLES: 'Hat ýazgylar',
+      NO_SUBTITLE: 'Hat ýazgysy ýok',
+      SUBTITLE_SIZE: 'Hat ýazgysynyň möçberi',
+      VIDEO_FILES: 'Wideonyň çeşmesi',
+      UPNEXT: 'Indiki...'
+    },
+
+    MESSAGES: {
+      SHARE_SOCKET: 'Siz ulgama gaýtadan girseňiz hem ýene-de şu sahypa ýüzlenersiňiz, ýöne bu gezek size tapawutly ID beriler. Eger siz başgalar bilen birbada meňzeş wideo görmek isleseňiz, siz öz ID-ňizi paýlaşyp bilersiňiz!',
+      FILE_MISSING: 'Bu maglumatda käbir kemçilikler ýüze çykdy. Siz ýalňyşlyk bilen bu faýlyň çeşmesini öçüren bolmagyňyz ahtimal.',
+      CODEC_PROBLEM: 'Wideo görkezijä täze wideo goşmaklyk üçin kemçilikler ýüze çykdy. Bu kemçilik, kod bilen baglanyşykly bolmagy ähtimal. Faýly HTML5 bilen sazlaşykly işlär ýaly edit, gaýtadan ýükläp görüň.',
+      WRONG_BASEPATH: 'Siz ýalňyş salga girizdiňiz',
+      FILE_IN_FS_NOT_FOUND: 'Siziň girizen salgyňyzdaky wideo tapylmady, salgyny dogrulap gaýtadan synanyşyp görüň.'
+    },
+    MANAGE_CONTENT: 'Wideolary dolandyr',
+    MANAGE_SUB_PROFILES: 'Şahsy maglumatlary dolandyr',
+    WHOS_WATCHING: 'Häzir kim görýär?',
+    ADD_SUB_PROFILE: 'Şahsy maglumat goş',
+    EDIT_BTN: 'Üýtget',
+    DONE_BTN: 'Tamamlandy',
+    SAVE_BTN: 'Ýatda sakla',
+    CREATE_BTN: 'Döret',
+    CANCEL_BTN: 'Bes et',
+    DELETE_BTN: 'Öçürmek',
+    ENTER_NAME: 'At giriz:',
+    EDIT_PROFILE: 'Şahsy maglumaty üýtget',
+    CREATE_PROFILE: 'Şahsy maglumat döret',
+    ADMIN: 'Administrator',
+    HELP: 'Kömek',
+    HELP_FAQ: 'KÖMEK / Iň köp soralýan soraglar',
+    PROFILE_SETTINGS: 'Ulanyjynyň Sazlamalary',
+    LOGOUT: 'Çykyş',
+    CHANGE_PASSWORD: 'Paroly üýtgetmek',
+    LANGUAGE_en: 'English',
+    LANGUAGE_cn: 'Chinese/中文',
+    LANGUAGE_ru: 'Русский/Russian',
+    LANGUAGE_de: 'Deutsch/German',
+    LANGUAGE_fr: 'Français/French',
+    LANGUAGE_es: 'Español/Spanish',
+    LANGUAGE_kr: '한국어/Korean',
+    LANGUAGE_nl: 'Nederlands/Dutch',
+    LANGUAGE_pt: 'Português/Portuguese',
+    LANGUAGE_ja: '日本語/Japanese',
+    LANGUAGE_it: 'Italiano/Italian',
+    LANGUAGE_da: 'Dansk/Danish',
+    LANGUAGE_ar: 'عربى/Arabic',
+    LANGUAGE_hu: 'Magyar/Hungarian',
+    LANGUAGE_tr: "Turkish/Türkçe",
+    LANGUAGE_tk: 'Turkmen/Türkmençe',
+    PROFIlE: {
+      USERNAME: 'Ulanyjy ady',
+      FULL_NAME: 'Doly ady',
+      LANGUAGE: 'Dili',
+      PAUSE_ON_CLICK: 'Ekrana basylanda wideony togtat',
+      FAVORITE_GENRES: 'Halanan žanrlar',
+      AMOUNT_OF_MEDIA_ENTRIES: 'Esasy sahypadaky ýüklenilmeli wideolaryň sany',
+      SAVE: 'Şahsy maglumatlary ýatda sakla',
+      PASS: 'Açar söz',
+      OLD_PASS: 'Öňki açar söz',
+      NEW_PASS: 'Häzirki açar söz',
```

Comment:
I think 'Açar söz' should be changed to 'Parol' as you used in `PASSWORD` field.

Reply:
Maybe you are probably correct, but we considered that 'Parol' is not turkmen word, that's why we used "Açar Söz"

Reply:
If so, please change places where you have used `Parol` to "Açar Söz".

Reply:
And you have translated `USERNAME` as 'Login' and 'Ulanyjy ady'. Please chose one of them to avoid misunderstandings.